aten/src/ATen/core/dispatch/OperatorEntry.cpp

```diff
@@ -35,7 +35,8 @@ OperatorEntry::OperatorEntry(OperatorName&& operator_name)
 
 namespace {
   void checkSchema(const OperatorName& name, const FunctionSchema& from_def, const std::string& from_def_debug, const FunctionSchema& inferred, const std::string& inferred_debug) {
-    c10::optional<std::string> schema_difference = findSchemaDifferences(from_def, inferred);
+    // TODO: figure out if we can just directly save real schema at def time
+    c10::optional<std::string> schema_difference = findSchemaDifferences(from_def.cloneWithRealTypes(), inferred);
```

Comment:
hmmm I wonder how much this impacts `import torch` time... Probably not too big of a deal though at least for mobile since they don't use the dispatcher now for registrations

Reply:
Yeah I'm not going to optimize this unless someone tells me it's a problem